cds-snc/covid-alert-app
# Add e2e region based tests

# Summary

Adds e2e detox tests for the region based screens (AllSetView, NoCode, NoExposureView).
In order to do this, DemoMode menu items were added to force the NoExposureViews.

# Test instructions

1. Run `yarn pre:test:ios` or `yarn pre:test:android` 
2. Run `yarn test:ios` or `yarn test:android`
3. Observe all tests pass

# Help requested

> Things that you (the submitter) want reviewers to pay very close attention to when they review this.

# Unresolved questions / Out of scope

> Are there any related issues or tangent features you consider out of scope for this issue that could be addressed in the future?

# Reviewer checklist

This is a suggested checklist of questions reviewers might ask during their review:
